src/About.tsx

```diff
@@ -140,6 +138,16 @@ export default function About() {
           </Grid>
         </Container>
 
+        <Typography variant="h5" gutterBottom>
+          Proper Crediting
+        </Typography>
+        <Typography>
+          Appropriate crediting is required to use the assets in this repository.
+          If you wish to use them, you must list all the authors that contributed to each asset you use, separately.
+          The recommended way of doing this is enabling credits mode on the main page, selecting all the assets you used, and clicking the "download credits" button to generate a single file with all the required credits.
```

Comment:
```suggestion
          The recommended way of doing this is enabling credits mode on the main page (under Search Options), selecting all the assets you used, and clicking the "download credits" button to generate a single file with all the required credits.
```
Since in the end we kept the button inside the search options box, I thought we could mention it here.